Client/Frontend/Browser/RecentlyClosedTabs.swift

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+public protocol RecentlyClosedTabs {
```

Comment:
Not sure there's much value in having a protocol here if we aren't mocking it or have multiple implementations. I'd just remove this.